tests/testthat/test-best_and_worst_cases.R

```diff
@@ -0,0 +1,26 @@
+test_that("Three `ep_products` with the same `benchmark` but with different
+          `emission_profile` will have `best_case` only for the low `emission_profile`
+          product and have `worst_case` only for the high `emission_profile` product", {
```

Comment:
Nice. Do NOT reflow test-titles. Test-titles do need to go in one line. (Once I cut a title to fit in 80 characters and that yielded a really mysterious bug. I reported it to testthat but Hadley (the maintainer) said test-titles are intended to be 1-line.)